src/selection.js

```diff
@@ -245,8 +245,35 @@ export default class Selection extends Cursor {
     this.setSelection()
   }
 
-  // Delete the contents inside the range. After that the selection will be a
-  // cursor.
+  // Delete the farest ancestor that is an exact selection
+  //
+  // @return Selection instance
+  deleteExactSurroundingMarkups () {
+    const ancestorMarkupTags = this.getAncestorTags(
+      (elem) => ['STRONG', 'EM'].includes(elem.nodeName)
+    ).reverse()
+    for (const ancestorMarkupTag of ancestorMarkupTags) {
+      if (this.isExactSelection(ancestorMarkupTag)) {
+        ancestorMarkupTag.remove()
+        break
+      }
+    }
+    return new Selection(this.host, this.range)
+  }
+
+  // Delete all the markups whose text is completely within the current selection.
+  //
+  // @return Selection instance
+  deleteContainedMarkupTags () {
+    const containedMarkupTags = this.getContainedTags(
+      (elem) => ['STRONG', 'EM'].includes(elem.nodeName)
```

Comment:
Why do you have here only an array of strong and em?

Reply:
I thought that those two are the only formattings that we can add on the text of an editable.

Reply:
I removed that filter so now it deletes all kind of tags